tests/phpunit/tests/option/registration.php

```diff
@@ -149,4 +149,28 @@ public function test_unregister_setting_removes_default() {
 
 		$this->assertFalse( has_filter( 'default_option_test_default', 'filter_default_option' ) );
 	}
+
+	/**
+	 * @ticket 57674
+	 *
+	 * @covers ::unregister_setting
+	 */
+	public function test_unregister_invalid_setting_does_not_raise_a_php_warning() {
```

Comment:
```suggestion
	public function test_unregister_invalid_setting_does_not_raise_php_notice_warning_or_error() {
```

Is a Notice on PHP 7. May be a fatal Error on PHP 9+.